.pylintrc

```diff
@@ -59,7 +59,9 @@ disable=fixme,
         invalid-name,
         too-many-lines,
         old-style-class,
-        superfluous-parens
+        superfluous-parens,
+        bad-continuation,
+        unexpected-keyword-arg
```

Comment:
why do we need these extra disabled rules?

Reply:
bad_continuation => is to workaround with `black`
unexpected-keyword-arg => is to handle python2 and 3 differences
e.g.  `self.proc.communicate(timeout=self.stop_timeout)` is python3 only and will have this in python2.
but we can also disable at code level.

Reply:
OK - `bad-continuation` sounds fine then, but yes let's keep `unexpected-keyword-arg` local where it's needed, that one could hide some confusing real issues if used globally.